Document/0x04f-Testing-Network-Communication.md

```diff
@@ -176,6 +176,7 @@ The interception proxy need to listen to the port specified in the port forwardi
 If not already done, install the CA certificates in your mobile device which will allow us to intercept HTTPS requests:
 
 - [Install the CA certificate of your interception proxy into your Android phone](https://support.portswigger.net/customer/portal/articles/1841102-installing-burp-s-ca-certificate-in-an-android-device "Installing Burp's CA Certificate in an Android Device").
+Note that starting at Android Nougat 7.0 (API LEVEL 24) the application no longer trusts a user supplied CA certificate unless specified in the app.Bypassing this security mesure will be addressed in the "Basic Security Testing" chapters.
```

Comment:
"at" => "with" Android Nougat
API LEVEL => API Level 
the application no longer trusts a user supplied CA => Isn't it the OS instead of the application?

Reply:
i believe it's the app because apps used to trust user CAs and now they don't

Reply:
But the Keystore on Android contains the CAs and this is where you will import the user CA. So if during the TLS Handshake the CA from a certificate is not known the OS will throw the error, not the app.

Reply:
In https://developer.android.com/training/articles/security-config you can see that indeed you can add CAs that are not trusted by the OS, but only for the app and are packed with the app :).